README.md

```diff
@@ -227,28 +227,46 @@ This tag helps you install or update Odoo modules without performing a full setu
 ansible-playbook playbook.yml --tags "only-modules"
 ```
 
-Community Roles
----------------
+Odoo modules
+------------
 
 #### Deploy
-To use community roles, you need to deploy this modules in the server. This role manage the modules deployment with `pip`.
+To use Odoo modules (from OCA or custom modules), you need to deploy modules packages to the server. This role manages the modules deployment with `pip`.
+In your inventory you can define up to two variables in order to add python packages to the target `requirements.txt` that will be generated.
 
-You can define a `requirements.txt` file to manage the modules and ensure the version installed:
+Define the `odoo_role_odoo_community_packages` variable to install packages not maintained by you. For example when you're deploying an Odoo instance that just requires OCA modules:
 
+```yml
+# inventory/group_vars/all.yml
+odoo_role_odoo_community_packages:
+  - odoo11-addon-contract==11.0.2.1.0
+  - odoo11-addon-contract-sale-invoicing==11.0.1.0.0
+  - odoo11-addon-contract-variable-qty-timesheet==11.0.1.0.0
+  - odoo11-addon-contract-variable-quantity==11.0.1.2.1
 ```
-# requirements.txt
-odoo11-addon-contract==11.0.2.1.0
-odoo11-addon-contract-sale-invoicing==11.0.1.0.0
-odoo11-addon-contract-variable-qty-timesheet==11.0.1.0.0
-odoo11-addon-contract-variable-quantity==11.0.1.2.1
+
+In some case you want to deploy different versions of the same module to different hosts.
+A typical case is when you have developed a custom module and need to deploy a packaged version of it to production but in local development environment you need to install it es editable.
+In this case you can define the `odoo_role_odoo_project_packages` variable:
+```yml
+# inventory/host_vars/production.yml
+odoo_role_odoo_project_packages:
+  - odoo14-my-custom-module==14.0.0.1.0
+```
+
+```yml
+# inventory/host_vars/development.yml
+odoo_role_odoo_project_packages:
+  - '-e file:///opt/odoo_modules/setup/my_custom_module'
 ```
 
-> The default the `requirements.txt` file path is `"{{ inventory_dir }}/../files/requirements.txt"`.
+For backward compatibility, the Ansible template will look first for a `files/requirements.txt` file in your inventory and use it as the source of the template.
+If you have a `files/requirements.txt` file defined in your inventory, the two variables just described will not take any effect.
```

Comment:
I think I'd feel more comfortable sticking with the requirements and adding the ability to uninstall specific modules on hosts marked for development, through the use of an Ansible task.

This way, we'd keep the advantages of the requirements file, and you could uninstall the pip package of the module you're working on in the dev environment.

However, haven't we already tackled this? :thinking:  It rings a bell that the addon path precedence in `odoo.conf` allowed the dev environment to pick up your folder instead of the pip package one.

Reply:
I guess you are referring to the CI issue, I've addressed this topic in my answer in other comment.